community/sysdig/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+name: sysdig
+version: 1.4.3
+appVersion: 0.89.4
+description: Sysdig Monitor and Secure agent
+keywords:
```

Comment:
We have some keywords that are required for better integration into our catalog; you can find out more here: [https://github.com/IBM/charts/blob/master/GUIDELINES.md#required-chart-keywords](https://github.com/IBM/charts/blob/master/GUIDELINES.md#required-chart-keywords)

Short version is you need at least ICP or IKS in your list to indicate support on one or both of them, and you need to select one or more of the architectures listed in that section for any of the architectures your chart supports.

There are also optional keywords that you can include that are listed in the guidelines as well, but they will not gate this PR.